samples/python/tutorial_code/core/file_input_output/file_input_output.py

```diff
@@ -0,0 +1,148 @@
+import cv2 as cv
+import numpy as np
```

Comment:
It is better to import numpy before cv2 (we have reports where this order of imports doesn't work)